src/pretix/plugins/paypal2/views.py

```diff
@@ -149,11 +149,20 @@ def post(self, request, *args, **kwargs):
         else:
             cart_total = get_cart_total(request)
             cart_payments = cart_session(request).get('payments', [])
-            for fee in get_fees(request.event, request, cart_total, None, cart_payments, get_cart(request)):
+            multi_use_cart_payments = [p for p in cart_payments if p.get('multi_use_supported')]
+            simulated_payments = multi_use_cart_payments + [{
+                'provider': 'paypal',
+                'multi_use_supported': False,
+                'min_value': prov.settings._total_min,
+                'max_value': prov.settings._total_max,
```

Comment:
```suggestion
                'min_value': None,
                'max_value': None,
```

min_value/max_value for a non-multi-use payment will get very confusing results in weird edge cases